@jorisvandenbossche do you maybe know what's necessary to fix the [pipeline error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=439247&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=519):

```
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/movingpandas_1641904570978/test_tmp/run_test.py", line 2, in <module>
    import movingpandas
  File "/home/conda/feedstock_root/build_artifacts/movingpandas_1641904570978/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.10/site-packages/movingpandas/__init__.py", line 27, in <module>
    from .tools._show_versions import show_versions  # noqa F401
ModuleNotFoundError: No module named 'movingpandas.tools'
Tests failed for movingpandas-0.9rc1-pyhd8ed1ab_0.tar.bz2 - moving package to /home/conda/feedstock_root/build_artifacts/broken
```

Local tests and CI on Github are fine: https://github.com/anitagraser/movingpandas/actions/workflows/tests.yaml

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,4 +1,4 @@
-{% set version = "0.9rc1" %}
+{% set version = "0.9rc2" %}
 
 package:
   name: movingpandas

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -6,7 +6,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/m/movingpandas/movingpandas-{{ version }}.tar.gz
-  sha256: e5d7cd568720ff00f02563706f9825632b201385496bc034dd144f8eddc1af54
+  sha256: eb60f87910d941859d4fd062cfbbf24decc78a8f20935031865d1ffbfe95129f
 
 build:
   number: 0

